docs/how_to/build_mxnet_with_intelmkl.md

```diff
@@ -0,0 +1,114 @@
+# Building MXNet with MKL-based optimizations:
+
+MXNet can be installed and used with several combinations of development 
+tools and libraries on a variety of platforms. This tutorial provides one 
+such recipe describing steps to build and install MXNet with Intel MKL on 
+an Ubuntu-based system, version 16.04.
+
+## Installing
+
+1. Update your operating system and install dependencies:
+
+   ```bash
+    $ sudo apt-get update && apt-get install -y build-essential git libopencv-dev curl gcc libatlas-base-dev python python-pip python-dev python-opencv graphviz python-scipy python-sklearn
```

Comment:
All other build instructions are in https://github.com/apache/incubator-mxnet/tree/master/docs/install, so please incorporate these instructions there.

Reply:
I got it. I will have an internal discussion regarding this.